os-config/ubuntu-credentials.robot

```diff
@@ -3,3 +3,4 @@ ${UBUNTU_USERNAME}=         ubuntu
 ${UBUNTU_PASSWORD}=         ubuntu
 ${UBUNTU_USER_PROMPT}=      ubuntu@3mdeb:~$
 ${UBUNTU_ROOT_PROMPT}=      root@3mdeb:/home/ubuntu#
+${UBUNTU_ROOTFS_PASSPHRASE}=    ubuntu
```

Comment:
@DaniilKl can't we just use `UBUNTU_PASSWORD`?

Reply:
Chenged to `UBUNTU_PASSWORD` here 391a08eb4061e6397fa1b30ecb6912b780e304c1